### ISSUE TITLE
Light theme while working with JupyterLab Dark theme messes up graphic-walker theme

### ISSUE THREAD

**Issue by PiotrBB** [Kanaries/graphic-walker#39]
It appears when `pyg` is used in `jupyter notebook` (JupyterLab 3.5.2) there are some inconsistencies in the theme. It works fine when I use JupyterLab Light, but with JupyterLab Dark it looks like below.
Can i somehow change `pyg` theme manually?

![image](https://user-images.githubusercontent.com/26751805/223073312-884c173e-3176-4515-a18f-1374e5bd9f9c.png)

**Comment by ObservedObserver** [comment]
It seems you are using a light theme of OS (media theme) and a dark theme of jupypter.

Graphic Walker now detects and syncs with your media theme. You can change your OS theme to dark to solve this problem.

Or I can add a parameter to allow users to determine the theme themself.

**Comment by PiotrBB** [comment]
Hi @ObservedObserver,
Since I'm using AWS Glue Docker image, I am not sure whether or not it is possible to change OS theme.
Nevertheless, I like the idea of having a parameter that could determine theme independently of current settings.

**Comment by ObservedObserver** [comment]
For now, Graphic Walker supports an extra param:

```ts
<GraphicWalker dark="light" />
<GraphicWalker dark="dark" />
<GraphicWalker dark="media" /> // auto detect OS theme
```

Wait for PyGWalker to update to the latest Graphic Walker and add new param in Python

**Comment by Asm-Def** [comment]
Please have a try with
```python
pyg.walk(df, dark='dark')  # 'class' | 'dark' | 'media'
```
The signature:
```raw
pyg.walk(
    df: 'pl.DataFrame | pd.DataFrame',
    gid: Union[int, str] = None,
    **kwargs,
)
kwargs:
    hideDataSourceConfig (bool, optional): Hide DataSource import and export button (True) or not (False). Default to True
    themeKey ('vega' | 'g2'): theme type.
    dark ('media' | 'light' | 'dark'): 'media': auto detect OS theme.
    return_html (bool, optional): Directly return a html string. Defaults to False.
```

This feature has been released in `0.1.4.8`.
https://github.com/Kanaries/pygwalker/pull/66
